webpack/assets/javascripts/react_app/components/common/AlertPanel/Alert.consts.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-throw-literal */
+const okClass = 'alert alert-success';
```

Comment:
imho no need to create a new component, can't we reuse Alert? similar changes to https://github.com/theforeman/foreman/pull/4795/files#diff-00d01709f729237a8f790e772209ea9c ?